### ISSUE TITLE
Windows: frameless window steals focus when calling maximize()

### ISSUE THREAD

**Issue by bpasero** [electron/electron#26512]
### Issue Details

* **Electron Version:**
  * 11.x
* **Operating System:**
  * Windows 10

### Expected Behavior
Calling `BrowserWindow.maximize()` does not focus the window.

### Actual Behavior
It focusses the window when `frame: false`.

### To Reproduce

Add this to a fiddle for the `electron-main` file:

```ts
// Modules to control application life and create native browser window
const {app, BrowserWindow} = require('electron')

// Keep a global reference of the window object, if you don't, the window will
// be closed automatically when the JavaScript object is garbage collected.
let mainWindow

function createWindow (maximize, focus) {
  // Create the browser window.
  mainWindow = new BrowserWindow({
    frame: false,
    width: 800,
    height: 600,
    webPreferences: {
      nodeIntegration: true
    }
  })

  // and load the index.html of the app.
  mainWindow.loadFile('index.html')

  if (maximize) {
    mainWindow.maximize();
  }

  if (focus) {
    mainWindow.focus();
  }

  // Open the DevTools.
  // mainWindow.webContents.openDevTools()

  // Emitted when the window is closed.
  mainWindow.on('closed', function () {
    // Dereference the window object, usually you would store windows
    // in an array if your app supports multi windows, this is the time
    // when you should delete the corresponding element.
    mainWindow = null
  })
}

// This method will be called when Electron has finished
// initialization and is ready to create browser windows.
// Some APIs can only be used after this event occurs.
app.on('ready', () => {
  createWindow(true, false);
  createWindow(false, true);
})

// Quit when all windows are closed.
app.on('window-all-closed', function () {
  // On OS X it is common for applications and their menu bar
  // to stay active until the user quits explicitly with Cmd + Q
  if (process.platform !== 'darwin') {
    app.quit()
  }
})

app.on('activate', function () {
  // On OS X it's common to re-create a window in the app when the
  // dock icon is clicked and there are no other windows open.
  if (mainWindow === null) {
    createWindow()
  }
})

// In this file you can include the rest of your app's specific main process
// code. You can also put them in separate files and require them here.
```

Start it and notice how the first window (that is maximized) gets focus, while the second should have it. Remove `frame: false` and see it work properly.

//cc @deepak1556

**Comment by ckerr** [comment]
gist version of @bpasero's repro: https://gist.github.com/ckerr/504dd32c94d7133b99fbd441874ad7d9

**Comment by zcbenz** [comment]
There are 2 low level call for maximization: `ShowWindow(hwnd, SW_MAXIMIZE)` and `SendMessage(hwnd, WM_SYSCOMMAND, SC_MAXIMIZE)`, I have tried both calls in a minimal Win32 app and both imply activating the window.

I also checked related messages, and there is no way to allow maximization without activating the window. 

So this is a Windows behavior that we can not work around, and I have to close this issue as won't fix.
